src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] lotteryBalls = new Ball[3];
+        for (int i = 0; i < lotteryBalls.length; i++) {
+            lotteryBalls[i] = new Lottery().getRandomBall();
```

Comment:
let's create Lottery object outside the loop, because now it will be created three times